charts/kyverno/templates/clusterrole.yaml

```diff
@@ -88,142 +22,224 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "kyverno.fullname" . }}:customresources
+  name: {{ template "kyverno.fullname" . }}:policies
   labels: {{ include "kyverno.labels" . | nindent 4 }}
     app: kyverno
 rules:
-# Kyverno CRs
 - apiGroups:
-  - '*'
+    - "kyverno.io"
   resources:
-  - policies
-  - policies/status
-  - clusterpolicies
-  - clusterpolicies/status
-  - policyreports
-  - policyreports/status
-  - clusterpolicyreports
-  - clusterpolicyreports/status
-  - generaterequests
-  - generaterequests/status
-  - reportchangerequests
-  - reportchangerequests/status
-  - clusterreportchangerequests
-  - clusterreportchangerequests/status
+    - policies
+    - policies/status
+    - clusterpolicies
+    - clusterpolicies/status
+    - policyreports
+    - policyreports/status
+    - clusterpolicyreports
+    - clusterpolicyreports/status
+    - generaterequests
+    - generaterequests/status
+    - reportchangerequests
+    - reportchangerequests/status
+    - clusterreportchangerequests
+    - clusterreportchangerequests/status
   verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-  - deletecollection
-- apiGroups:
-  - 'apiextensions.k8s.io'
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - delete
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+    - deletecollection
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "kyverno.fullname" . }}:policycontroller
+  name: {{ template "kyverno.fullname" . }}:view
   labels: {{ include "kyverno.labels" . | nindent 4 }}
     app: kyverno
 rules:
-# background processing, identify all existing resources
 - apiGroups:
   - '*'
   resources:
   - '*'
   verbs:
   - get
   - list
-  - update
   - watch
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "kyverno.fullname" . }}:generatecontroller
+  name: {{ template "kyverno.fullname" . }}:mutate
```

Comment:
Can you help me understand this role? Where would we use it?

Reply:
We do not. I removed it from the YAMLs but did not clean up here!